<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>remoteAssist</class>
 <widget class="QDialog" name="remoteAssist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remote Assistant</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Active Interface:</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0" colspan="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Residence</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QToolButton" name="loadID">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/Graphics/forward</normaloff>:/Graphics/forward</iconset>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="3">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Tel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QComboBox" name="_sex">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Male</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Female</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QComboBox" name="_martital">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Single</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Married</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Widdowed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Devorced</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="5">
      <widget class="QLineEdit" name="_occu">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>visitor occupation</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="3">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>occupation</string>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QLineEdit" name="_pob">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>visitor place of birth </string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Place of birth</string>
       </property>
      </widget>
     </item>
     <item row="9" column="5">
      <widget class="QComboBox" name="_visitType">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>New</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Follow 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Follow 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Follow 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Follow 4 </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Free</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QLineEdit" name="_residence">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>visitor address</string>
       </property>
      </widget>
     </item>
     <item row="8" column="5">
      <widget class="QLineEdit" name="_tel">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>visitor contact number</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="3">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Visit Type</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QSpinBox" name="_age">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="_name">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>visitor name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Age</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLineEdit" name="_id">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>visitor id number</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Marital</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QToolButton" name="loadName">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/Graphics/forward</normaloff>:/Graphics/forward</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Sex</string>
       </property>
      </widget>
     </item>
     <item row="9" column="7">
      <widget class="QToolButton" name="reload">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/Graphics/refresh</normaloff>:/Graphics/refresh</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QToolButton" name="searchByID">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/Graphics/search</normaloff>:/Graphics/search</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QToolButton" name="searchByName">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/Graphics/search</normaloff>:/Graphics/search</iconset>
       </property>
      </widget>
     </item>
     <item row="8" column="6">
      <widget class="QToolButton" name="searchByTel">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/Graphics/search</normaloff>:/Graphics/search</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="id_check">
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="name_check">
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="interfaces"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTableView" name="visitorsTableView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideLeft</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
